conf/application.conf

```diff
@@ -72,19 +72,13 @@ controllers {
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
   enabled = true
```

Comment:
do you need this to be enabled? Could probably just delete the whole `metrics` stanza